df.viewscreen.xml

```diff
@@ -2168,7 +2168,15 @@
             <enum-item name='MentalTraits'/>
             <enum-item name='Background'/>
         </enum>
-        <stl-vector name='unk_vec1' since='v0.42.01'/>
+        <stl-vector name='races_info' since='v0.42.01'>
+            <pointer>
+                    <int32_t name='race' type-name='int32_t' ref-target='creature_raw'/>
```

Comment:
The type-name here isn't needed, and I think it's the cause of [this on Buildmaster](https://buildmaster.lubar.me/executions/execution-details?executionId=71165#S12.87):
```
E: NameError: ./hack/ruby/ruby-autogen-gcc.rb:59640: uninitialized constant DFHack::ViewscreenSetupadventurest::Int32T
 (eval)
 ./hack/ruby/ruby-autogen-defs.rb:44:in `field'
 ./hack/ruby/ruby-autogen-gcc.rb:59639
 ./hack/ruby/ruby-autogen-defs.rb:126:in `class_eval'
 ./hack/ruby/ruby-autogen-defs.rb:126:in `compound'
 ./hack/ruby/ruby-autogen-gcc.rb:59636
 ./hack/ruby/ruby-autogen-defs.rb:75:in `pointer'
 ./hack/ruby/ruby-autogen-gcc.rb:59635
```

(Also, this line through `</pointer>` are indented too far)